### PR TITLE
Authentiation

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
- Authentication with Test Login Credentials

Files to review:
- AuthContext
- Login
- Userdetailsreducer

### CODE REVIEW

File: src/Pages/Login.jsx [hunk 1]
```diff
@@ -1,9 +1,105 @@
...
+    loginHandler();
+  };
+
+  const loginHandler = async () => {
+    try {
+      const loginResp = await axios.request({
+        method: "post",
+        url: "/api/auth/login",
+        data: {
+          email: credentials.email,
```
Comment (srishtigautam21) [inline src/Pages/Login.jsx:41]: You can move all of this API call in auth context

**Review by srishtigautam21 — APPROVED** [review]
The code is clean and good to merge

**Comment by vivekgudipudi** [comment]
image

- here I have added item to cart from both wishlist page and productlisting page, in this case try to increment the cart rather than adding similar cards

**Review by vivekgudipudi — APPROVED** [review]
try to implement the changes commented, 
code is clean and well structured, good to merge

**Review by Varu98 — APPROVED** [review]
Clean UI 👍🏻

**Pull request merged**